acapy_wallet_upgrade/strategies.py

```diff
@@ -135,8 +135,9 @@ async def update_items(
         wallet: Wallet,
         indy_key: dict,
         profile_key: dict,
+        limit: int,
```

Comment:
Rather than making this be a parameter and then passing it to the nested methods, I think limit makes sense to be a member of the Strategy class (`self.limit`). This will reduce the number of changes we have to make in this PR as well as making it clear that we're using the same limit across all the different types of items.